<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProviders</class>
 <widget class="QDialog" name="DialogProviders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fournisseurs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/app/providers</normaloff>:/app/providers</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_add">
          <property name="text">
           <string>Ajouter</string>
          </property>
          <property name="icon">
           <iconset resource="../ressources.qrc">
            <normaloff>:/app/insert</normaloff>:/app/insert</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_edit">
          <property name="text">
           <string>Modifier</string>
          </property>
          <property name="icon">
           <iconset resource="../ressources.qrc">
            <normaloff>:/app/Edit</normaloff>:/app/Edit</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete">
          <property name="text">
           <string>Supprimer</string>
          </property>
          <property name="icon">
           <iconset resource="../ressources.qrc">
            <normaloff>:/app/remove</normaloff>:/app/remove</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_providers"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_add</tabstop>
  <tabstop>pushButton_edit</tabstop>
  <tabstop>tableWidget_providers</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
